kedro/extras/datasets/json/json_dataset.py

```diff
@@ -28,17 +28,13 @@ class JSONDataSet(AbstractVersionedDataSet[Any, Any]):
         >>> json_dataset:
         >>>   type: json.JSONDataSet
         >>>   filepath: data/01_raw/location.json
-        >>>   load_args:
```

Comment:
JSON Dataset does not support the `load_args` parameter: https://kedro.readthedocs.io/en/stable/kedro.extras.datasets.json.JSONDataSet.html#kedro.extras.datasets.json.JSONDataSet.__init__

Reply:
Great catch!